### PR TITLE
add the opportunity for PageView (same behavior as TabView) 

### PR DESCRIPTION
**Pull request by spoeck**
Thanks for your grate work! Very nice design!

Now you can use a PageController. Just give an instance to the FancyBottomNavigation and to a PageView and it works. You can checkout the example.

### CODE REVIEW

**Comment by GeertJohan** [comment]
This is really nice, thanks @spoeck, I'm already using this in my fork.

File: lib/fancy_bottom_navigation.dart [hunk 1]
```diff
@@ -246,12 +255,41 @@ class FancyBottomNavigationState extends State<FancyBottomNavigation>
...
+      _initAnimationAndStart(0);
+
+      setState(() {
+        currentSelected = page;
+      });
+    }
+  }
+
+  void setPageOffset(double page) {
+    print("$page");
```
Comment (GeertJohan) [inline lib/fancy_bottom_navigation.dart]: This print() is a bit spammy

**Review by AliRn76 — APPROVED** [review]
You should accept this pull request , this was very useful :)

**Comment by clragon** [comment]
I would love to have this in the package, but I'm not sure if the author is still active.

**Pull request merged**